From 3608628ffb0df690026e652eba9580458c479495 Mon Sep 17 00:00:00 2001
From: slin14 <shuicheng.lin@intel.com>
Date: Thu, 16 Aug 2018 02:44:43 +0800
Subject: [PATCH] Do not pass CFLAGS to gcc while building native tool

Do not pass CFLAGS/LDFLAGS to gcc while building native tool, The
needed flags has been passed by xxx_CFLAGS=$(CFLAGS_FOR_BUILD).
---
 tools/locktest/Makefile.am | 2 ++
 tools/rpcgen/Makefile.am   | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/tools/locktest/Makefile.am b/tools/locktest/Makefile.am
index 3156815..1729fd1 100644
--- a/tools/locktest/Makefile.am
+++ b/tools/locktest/Makefile.am
@@ -1,6 +1,8 @@
 ## Process this file with automake to produce Makefile.in
 
 CC=$(CC_FOR_BUILD)
+CFLAGS=
+LDFLAGS=
 LIBTOOL = @LIBTOOL@ --tag=CC
 
 noinst_PROGRAMS = testlk
diff --git a/tools/rpcgen/Makefile.am b/tools/rpcgen/Makefile.am
index 8a9ec89..8bacdaa 100644
--- a/tools/rpcgen/Makefile.am
+++ b/tools/rpcgen/Makefile.am
@@ -1,6 +1,8 @@
 ## Process this file with automake to produce Makefile.in
 
 CC=$(CC_FOR_BUILD)
+CFLAGS=
+LDFLAGS=
 LIBTOOL = @LIBTOOL@ --tag=CC
 
 noinst_PROGRAMS = rpcgen
-- 
2.7.4

